marlowe-runtime/marlowe-runtime.cabal

```diff
@@ -162,6 +162,73 @@ library config
     , text
     , split
 
+library web
+  import: lang
+  hs-source-dirs: web
+  exposed-modules:
+    Language.Marlowe.Runtime.Web
+  other-modules:
+    Language.Marlowe.Runtime.Web.API
+    Language.Marlowe.Runtime.Web.Types
+  build-depends:
+      base >= 4.9 && < 5
+    , aeson
+    , base16
+    , bytestring
+    , containers
+    , lens
+    , marlowe
+    , marlowe-runtime
+    , openapi3
+    , servant
+    , servant-pagination
+    , servant-server
+    , servant-openapi3
+    , text
+
+library web-server
+  import: lang
+  hs-source-dirs: web-server
+  exposed-modules:
+    Language.Marlowe.Runtime.Web.Server
+  other-modules:
+    Language.Marlowe.Runtime.Web.Server.DTO
+    Language.Marlowe.Runtime.Web.Server.Flags
+    Language.Marlowe.Runtime.Web.Server.Monad
+    Language.Marlowe.Runtime.Web.Server.OpenAPI
+    Language.Marlowe.Runtime.Web.Server.ContractHeaderIndexer
+    Language.Marlowe.Runtime.Web.Server.REST
+    Language.Marlowe.Runtime.Web.Server.REST.Contracts
+  build-depends:
+      base >= 4.9 && < 5
+    , aeson
+    , async
+    , bytestring
+    , cardano-api
+    , containers
+    , eventuo11y
+    , eventuo11y-dsl
+    , eventuo11y-json
+    , eventuo11y-batteries
+    , exceptions
+    , lens
+    , marlowe-runtime:web
```

Comment:
Is there a difference between doing `marlowe-runtime:web` vs `web`? Or is it just to make sure cabal explicitly looks in this project rather than on the web?

Reply:
`marlowe-runtime:web` is the only way to refer to this sub-library